src/components/services-view/services-view.controller.ts

```diff
@@ -65,12 +65,17 @@ export class ServicesViewController implements angular.IController {
     }
 
     this.ctrl.currentFilter = category;
-    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || 'all');
+    this.ctrl.currentSubFilter = (this.ctrl.subCategories.length === 1) ? this.ctrl.subCategories[0].id : (subCategory || null);
```

Comment:
Thanks, @dtaylor113.  Can you also fix the unit test?  Like @jeff-phillips-18 and @spadgett, they won't run for me.  :(